pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1892,21 +1960,37 @@ ASTSelectStatement SelectStatement() :
 ASTUpdateStatement UpdateStatement() :
 {}
 {
-    <UPDATE> SqlExpression()
+    <UPDATE> DMLTableExpressionClause() [ LOOKAHEAD(2) TableAlias() ]
     UpdateSetClause()
     [ WhereClause() ]
     { return jjtThis; }
 }
 
+ASTDMLTableExpressionClause DMLTableExpressionClause() :
+{}
+{
+  (
+    LOOKAHEAD(2) TableName()
```

Comment:
isn't this case covered by the third one? do we need to explicitly include it here to avoid an issue?